packages/components/src/ClickableStyle/ClickableStyle.stories.tsx

```diff
@@ -17,5 +19,72 @@ ClickableStyleAsAnATag.args = {
   color: "brand",
   variant: "flat",
   size: "medium",
-  children: "ClickableStyle As An A Tag",
+  children: "ClickableStyle",
+};
+
+export const ClickableStyleWithIconOnLeftSide = Template.bind(null);
+ClickableStyleWithIconOnLeftSide.args = {
+  as: "a",
+  color: "brand",
+  variant: "flat",
+  size: "medium",
+  children: "ClickableStyle",
+  icon: {
+    iconElement: <ArrowBackRoundedIcon purpose="decorative" />,
+    iconHorizontalMargin: "-4px",
+  },
+};
```

Comment:
Kinda wanna add another grid to the button stories so we can see the icons on the left, right, and along on all the variants. I put these here because I didn't want to get too bogged down in thinking about the button stories' organization.